doc/release-notes/5.0-release-notes.md

```diff
@@ -0,0 +1,313 @@
+
+Please note that we are not aware of any bugs in the current versions of Dataverse that would result in damage to users' files. But you may have some legacy files in your archive that were affected by some issue in the past, or perhaps affected by something outside Dataverse, so we are adding this feature out of abundance of caution. An example of a problem we've experienced in the early versions of Dataverse was a possible scenario where a user actually attempted to delete a Draft file from an unpublished version, where the database transaction would fail for whatever reason, but only after the physical file had already been deleted from the filesystem. Thus resulting in a datafile entry remaining in the dataset, but with the corresponding physical file missing. (the fix for this case, since the user wanted to delete the file in the first place, is simply to confirm it and purge the datafile entity from the database).
+
+### The setting :PIDAsynchRegFileCount is deprecated as of v5.0
+
+It used to specify the number of datafiles in the dataset to warrant adding a lock during publishing. As of v5.0 all datasets get locked for the duration of the publishing process. The setting will be ignored if present.
+
+### Location Changes for Related Projects
+
+The dataverse-ansible and dataverse-previewers repositories have been moved to the GDCC Organization on GitHub. If you have been referencing the dataverse-ansible repository from IQSS and the dataverse-previewers from QDR, please instead use them from their new locations:
+
+<https://github.com/GlobalDataverseCommunityConsortium/dataverse-ansible>
+<https://github.com/GlobalDataverseCommunityConsortium/dataverse-previewers>
+
+### Harvesting Improvements
+
+Many updates have been made to address common Harvesting failures. You may see Harvests complete more often and have a higher success rate on a dataset-by-dataset basis.
+
+### New JVM Options and Database Settings
+
+Several new JVM options and DB Settings have been added in this release. More documentation about each of these settings can be found in the Configuration section of the [Installation Guide](http://guides.dataverse.org/en/5.0/installation/config.rst).
+
+#### New JVM Options
+
+- :doi.baseurlstringnext: Set with a value of "https://api.datacite.org" for production environments and "https://api.test.datacite.org" for test environments. Must be set if you are using DataCite as your DOI provider.
+- :dataverse.useripaddresssourceheader: If set, specifies an HTTP Header such as X-Forwarded-For to use to retrieve the user's IP address. This setting is useful in cases such as running Dataverse behind load balancers where the default option of getting the Remote Address from the servlet isn't correct (e.g. it would be the load balancer IP address). Note that unless your installation always sets the header you configure here, this could be used as a way to spoof the user's address.
```

Comment:
Just noting that this sounds a little scary:

"Note that unless your installation always sets the header you configure here, this could be used as a way to spoof the user's address."

Maybe we could end with "If you don't need this setting, don't set it."

Reply:
Or 'see guide for proper use and details about the security concern.'

Reply:
Ah, I didn't realize there's more detail in the guides about the security concern. Sure, mentioning this would be good. Or we could remove the scary line and let people discover it in the guides. Less is more.